app/assets/stylesheets/modules/record-metadata-section.scss

```diff
@@ -60,3 +60,7 @@
 ul.toc {
   padding-left: 1.2em;
 }
+
+.content-advice {
+  background-color: #FEC51D1A;
```

Comment:
We've been collecting these palette colors in [sul-variables](https://github.com/sul-dlss/SearchWorks/blob/main/app/assets/stylesheets/sul-variables.scss).